.circleci/config.yml

```diff
@@ -71,8 +71,8 @@ jobs:
       - checkout
       - run: Packaging/amiga/prep.sh
       - run: mkdir build
-      - run: cd build && PKG_CONFIG_PATH=/opt/m68k-amigaos/lib/pkgconfig/:/opt/m68k-amigaos/share/pkgconfig/ cmake -DM68K_CPU=68040 -DM68K_FPU=hard -DM68K_COMMON="-s -ffast-math -O3 -noixemul -fpermissive -DAMIGA -fbbb=abcdef" ..
-      - run: cd build && cmake --build . --config Release -- -j$(nproc)
+      - run: cd build && PKG_CONFIG_PATH=/opt/m68k-amigaos/lib/pkgconfig/:/opt/m68k-amigaos/share/pkgconfig/ cmake -DM68K_CPU=68040 -DM68K_FPU=hard -DM68K_COMMON="-s -ffast-math -O3 -noixemul -DBIG_ENDIAN -DAMIGA -fpermissive -fbbb=abcdef" ..
+      - run: cd build && make -j$(nproc)
```

Comment:
Why make?

Reply:
It's what we do on all other builds, it's the default generator and it's simple to read and understand for most users that want to try and compile them selfs.

Reply:
Well, so is cmake --build, and on systems with msbuild, it will default to that instead of GNU Make.

Reply:
Noted I might update all the guides soon and will probably apply this change in the process, thanks

Reply:
Everything has now been switched to recomend cmake --build instead https://github.com/diasurgical/devilutionX/commit/ebe846d6ef279b7977dc87e0ef0e43ac4661e891